components/form/Internal/Validate/Rule/RuleValidateHelper.cs

```diff
@@ -0,0 +1,407 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace AntDesign.Internal
+{
+    internal class RuleValidateHelper
```

Comment:
I cannot find any non-static methods, so probably the class should also be static?